server/routes/bookAVisit.ts

```diff
@@ -392,33 +393,14 @@ export default function routes(
       visitSessionData.visitorSupport
     )
 
-    if (!req.session.visitSessionData.visitReference) {
```

Comment:
Took this block out as it wouldn't get hit now that we have the session check middleware checking before it gets here